lib/perfectqueue/multiprocess/thread_processor.rb

```diff
@@ -121,7 +121,7 @@ def run_loop
       end
 
       def process(task)
-        @log.info "acquired task task=#{task.key} id=#{@processor_id}: #{task.inspect}"
+        @log.info "acquired task task=#{task.key} id=#{@processor_id}"
```

Comment:
Sometimes task data is important for debugging. How about adding task.inspect as debug level log? /cc @nurse 
```
@log.info "acquired task task=#{task.key} id=#{@processor_id}"
@log.debug "task data: #{task.inspect}"
```